cli/samples/default-config.yaml

```diff
@@ -0,0 +1,38 @@
+# ------ DO NOT EDIT ------------------
+# These are the default values used for `gkekitctl create` when 
+# the user does not supply their own config.yml. Note that the user 
+# *does* have to provide their GCP project ID at runtime.  
+terraformState: local  
+clustersProjectId: "my-project"
+governanceProjectId: "my-project"
+region: "us-central1x"
```

Comment:
Is that x a typo?

Reply:
yes I was testing the validation, will fix!

Reply:
Awesome! Thanks Megan!